src/routes/gallery.tsx

```diff
@@ -21,33 +21,118 @@ export const app = new Frog<HonoEnv>({})
 //       <Button.Link href="https://kodadot.xyz">kodadot</Button.Link>
 //     ],
 //   })
-// })  
+// })
+function UnverifiedImage({ imageUrl }: {imageUrl: string}) {
```

Comment:
![Screenshot 2024-04-30 at 10 15 34](https://github.com/vikiival/based-frog-frame/assets/22471030/acafc497-b941-4d17-99a4-44f77092e9cd)


This is not very nice but can be updated by us later 

cc @exezbcz

Reply:
yeah can be updated later, it functional